packages/pwa/app/commerce-api/utils.js

```diff
@@ -278,3 +285,34 @@ export const convertSnakeCaseToSentenceCase = (text) => {
  * Usually used as default for event handlers.
  */
 export const noop = () => {}
+
+/**
+ * Creates a http/https Node.js Agent
+ * @param protocol
+ * @returns {Promise<*>}
+ */
+export const createAgent = async (protocol) => {
```

Comment:
To persist connections server side, we'll need to implement this as a singleton, that is, repeated calls should return the same instance.